lib/Send/AHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright 2024 Anna Larch <anna.larch@gmx.net>
+ *
+ * @author Anna Larch <anna.larch@gmx.net>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace OCA\Mail\Send;
+
+use OCA\Mail\Account;
+use OCA\Mail\Db\LocalMessage;
+
+abstract class AHandler {
+
+	protected AHandler|null $next = null;
+
+	public function __construct() {
+	}
```

Comment:
unused -> drop. an empty constructor only makes sense for public API and where we have to avoid the fragile baseclass problem. we can always adjust all sub classes if a constructor is needed here in the future.